project_timeline/views/project_task_view.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2016-2017 Tecnativa - Pedro M. Baeza
+     Copyright 2017 Tecnativa - Carlos Dauden
+     Copyright 2017 Doscaal - Alexandre Moreau
+     Copyright 2021 Open Source Integrators - Daniel Reis
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+
+    <record id="project_task_timeline" model="ir.ui.view">
+        <field name="model">project.task</field>
+        <field name="type">timeline</field>
+        <field name="arch" type="xml">
+            <timeline
+                date_start="date_start"
+                date_stop="date_end"
+                default_group_by="project_id"
+                event_open_popup="true"
+                colors="white: user_ids == []; #2ecb71: kanban_state == 'done'; #ec7063: kanban_state == 'blocked'"
+            >
+                <field name="user_ids" />
+                <field name="planned_hours" />
+                <templates>
+                    <t t-name="timeline-item">
+                        <div class="o_project_timeline_item">
+                            <img
+                                t-if="record.user_ids"
+                                t-attf-src="/web/image/res.users/#{record.user_ids[0][0]}/image_128/16x16"
```

Comment:
I had to change to user_ids[0] to give ID or no image displayed:

`t-attf-src="/web/image/res.users/#{record.user_ids[0]}/image_128/16x16"`

Reply:
You can display all user_ids images with this:

`<t t-foreach="record.user_ids" t-as="user_id">
    <img
        t-attf-src="/web/image/res.users/#{user_id}/image_128/16x16"
        width="16"
        height="16"
        class="mr8"
        alt="User"
    />
</t>`